### PR TITLE
docs(tutorial): umami

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
tutorial umami

### CODE REVIEW

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  paragraph: In this tutorial you will learn how to deploy umami web analytics on Serverless Containers.
+tags: umami serverless containers 
+  - compute
+  - serverless
+dates:
+  validation: 2022-04-21
+  posted: 2021-04-21
+---
+
+umami is a simple, easy to use, self-hosted web analytics solution. It provides a easier-to-use, open-source, and pprivacy-focused alternative paid solutions such as Google Analytics.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:17]:
```suggestion
Umami is a simple, easy to use, self-hosted web analytics solution. It provides an easy-to-use, open-source, and privacy-focused alternative to paid solutions such as Google Analytics. 
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    ```md
+    psql -h <postgresql-ip>:<port> -U username -d databasename -f sql/schema.postgresql.sql
+    ```
+
+## Preparating the Container Registry
+
+1. Sign in to your namespace in your terminal:
+    ```
+    docker login rg.fr-par.scw.cloud/<umami-namespace> -u nologin -p $SCW_SECRET_TOKEN
+    ```
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:51]:
```suggestion
    ```

```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    docker push rg.fr-par.scw.cloud/<umami-namespace>/umami:postgresql-latest
+    ```
+
+5. Click the **Namespace Settings** tab in the **Container Registry** section of the Scaleway console and make sure your namespace privacy policy is set to **Public**. 
+    <Lightbox src="scaleway-umami-namespace-public.webp" alt="" />
+
+## Deploying the Serverless Container
+
+1. Select the Serverless Containers namespace you want to use to deploy the application from the **Serverless Containers** section in the Scaleway console. If you do not have a Serverless Containers namespace yet, click **+ Create a namespace** to create a new one.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:76]:
```suggestion
1. Select the namespace you want to use to deploy the application in the **Serverless Containers** section of the Scaleway console. If you do not have a Serverless Containers namespace yet, click **+ Create a namespace** to create a new one.
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    <Lightbox src="scalway-serverless-containers-namesp-list.webp" alt="" />
+
+2. Click **+ Deploy a Container**. The container deployment wizard displays.
+
+3. Choose the namespace you pushed the image to and select the `umami:postgresql-latest` image. Set the value for the port to `3000`.
+
+2. Enter a name for your container. Optionally you can enter a container description.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:83]:
```suggestion
2. Enter a name for your container. Optionally, you can enter a container description. 
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+1. Select the Serverless Containers namespace you want to use to deploy the application from the **Serverless Containers** section in the Scaleway console. If you do not have a Serverless Containers namespace yet, click **+ Create a namespace** to create a new one.
+    <Lightbox src="scalway-serverless-containers-namesp-list.webp" alt="" />
+
+2. Click **+ Deploy a Container**. The container deployment wizard displays.
+
+3. Choose the namespace you pushed the image to and select the `umami:postgresql-latest` image. Set the value for the port to `3000`.
+
+2. Enter a name for your container. Optionally you can enter a container description. 
+
+3. Select the resources you want to allocate to the container. The resouces to allocate are depending on the intended load of your application. For the sake of this tutorial choose **128 MB - 70 mCPU**.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:85]:
```suggestion
3. Select the resources you want to allocate to the container. The resources to allocate depend on the intended load of your application. In this tutorial we use **128 MB - 70 mCPU**.
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+4. Configure the minimum and maximum values to autoscale the number of available instances for your container. 
+
+7. Set the Secrets. Secrets are an extra-secure type of environment variables which are injected into your container and stored securely:
+    <Lightbox src="scaleway-serverless-secrets.webp" alt="" />
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:90]:
```suggestion

    <Lightbox src="scaleway-serverless-secrets.webp" alt="" />

```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+8. Verify that the container privacy policy is set to **Public**.
+
+9. Click **Deploy a Container**.
+
+## Accessing the umami dashboard
+
+1. Click the **Container Settings** tab, to display container information. 
+
+2. Copy the URL displayed at **Container Endpoint** and paste in in a web browser. The umami login screen displays.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:108]:
```suggestion
2. Copy the URL displayed for **Container Endpoint** and paste in in a web browser. The umami login screen displays.
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+    * User: `admin`
+    * Password: `umami`
+
+    <Lightbox src="scaleway-umami-dashboard.webp" alt="" />
+
+    The umami dashboard displays and you can add websites to monitor. For more information about umami refer to the [official documentation](https://umami.is/docs/).
+
+## Customizing the endpoint name (otionally)
+
+Your Serverless Functions comes with a default entpoint name. You can customize this to a name of your choice using the **Endpoints** feature.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:121]:
```suggestion
Your Serverless Functions come with a default endpoint name. You can customize this to a name of your choice using the **Endpoints** feature.
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+1. Click the **Entpoints** tab. The endpoint information displays.
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:123]:
```suggestion
1. Click the **Endpoints** tab. The endpoint information displays. 
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+## Customizing the endpoint name (otionally)
+
+Your Serverless Functions comes with a default entpoint name. You can customize this to a name of your choice using the **Endpoints** feature.
+
+1. Click the **Entpoints** tab. The endpoint information displays. 
+
+2. Click **+ Add new Endpoints**. A pop up displays. 
+
+3. Enter the name of your endpoint. You can use any domain or subdomain that you own. Click **Add Endpoint** to validate.
+    <Lightbox src="scaleway-serverless-endpoint.webp" alt="" />
```
Comment (ldecarvalho-doc) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:128]:
```suggestion

    <Lightbox src="scaleway-serverless-endpoint.webp" alt="" />

```

**Review by ldecarvalho-doc — APPROVED** [review]

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+    git clone https://github.com/mikecao/umami.git
+    cd umami
+    ```
+
+3. Install the required tables from the script shipped with umami: 
+    ```md
+    psql -h <postgresql-ip>:<port> -U username -d databasename -f sql/schema.postgresql.sql
+    ```
+
+## Preparating the Container Registry
```
Comment (RoRoJ) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:46]:
```suggestion
## Preparing the Container Registry
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+1. Sign in to your namespace in your terminal:
+    ```
+    docker login rg.fr-par.scw.cloud/<umami-namespace> -u nologin -p $SCW_SECRET_TOKEN
+    ```
+    <Message type="important">
+
+    In order to run this example, you must first create a Secret Token from the credentials page, and replace the expression `$SCW_SECRET_TOKEN` with the created secret.
```
Comment (RoRoJ) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:54]:
```suggestion
    In order to run this example, you must first create a secret token from the credentials page, and replace the expression `$SCW_SECRET_TOKEN` with the created token.
```

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+    ```
+3. Tag the local container 
+    ```
+    docker tag ghcr.io/mikecao/umami:postgresql-latest rg.fr-par.scw.cloud/<umami-namespace>/umami:postgresql-latest
+    ```
+4. Push the local container to your Container Registry namespace:
+    ```
+    docker push rg.fr-par.scw.cloud/<umami-namespace>/umami:postgresql-latest
+    ```
+
```
Comment (RoRoJ) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:70]: Extra line break?

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+3. Choose the namespace you pushed the image to and select the `umami:postgresql-latest` image. Set the value for the port to `3000`.
+
+2. Enter a name for your container. Optionally you can enter a container description. 
+
+3. Select the resources you want to allocate to the container. The resouces to allocate are depending on the intended load of your application. For the sake of this tutorial choose **128 MB - 70 mCPU**.
+
+4. Configure the minimum and maximum values to autoscale the number of available instances for your container. 
+
+7. Set the Secrets. Secrets are an extra-secure type of environment variables which are injected into your container and stored securely:
```
Comment (RoRoJ) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:89]: Steps go from 4 to 7

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+## Deploying the Serverless Container
+
+1. Select the Serverless Containers namespace you want to use to deploy the application from the **Serverless Containers** section in the Scaleway console. If you do not have a Serverless Containers namespace yet, click **+ Create a namespace** to create a new one.
+    <Lightbox src="scaleway-serverless-containers-namesp-list.webp" alt="" />
+
+2. Click **+ Deploy a Container**. The container deployment wizard displays.
+
+3. Choose the namespace you pushed the image to and select the `umami:postgresql-latest` image. Set the value for the port to `3000`.
+
+2. Enter a name for your container. Optionally you can enter a container description.
```
Comment (RoRoJ) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:83]: Steps go from 3 to 2

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+
+8. Verify that the container privacy policy is set to **Public**.
+
+9. Click **Deploy a Container**.
+
+## Accessing the umami dashboard
+
+1. Click the **Container Settings** tab, to display container information. 
+
+2. Copy the URL displayed at **Container Endpoint** and paste in in a web browser. The umami login screen displays.
```
Comment (RoRoJ) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:108]:
```suggestion
2. Copy the URL displayed for **Container Endpoint** and paste it into a web browser. The umami login screen displays.
```

**Pull request merged**